src/OpenColorIO/fileformats/FileFormatSpi3D.cpp

```diff
@@ -172,8 +176,20 @@ CachedFileRcPtr LocalFileFormat::read(
             lutArray[index+0] = redValue;
             lutArray[index+1] = greenValue;
             lutArray[index+2] = blueValue;
-
-            entriesRemaining--;
+            if(ifDuplicated[index] == false){
```

Comment:
Minor:  Please put the "{" on the next line to follow the coding style.  Same for line 183.  Also please put a space between "if" and "(".

Reply:
Thanks for your reminding！I have changed the code style.